pkg/sync/v10/client.go

```diff
@@ -7,12 +7,11 @@ import (
 
 	"github.com/sirupsen/logrus"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
```

Comment:
Changes to pkg/sync/v* (except v13) are knock-on effects of the revendor to origin's kubernetes tree.